### DISCUSSION TITLE
Using Prisma with Vue 3

### DISCUSSION THREAD

**Discussion by murugappanrm** [prisma/prisma#16123] [Q&A] [answered]
Hi,

I have seen YouTube videos on using Prisma in Nuxt app but i could not find any Prisma/Vue integration and usage. Has anyone used Prisma with pure vue 3 framework?

Help is most appreciated.

**Comment by pantharshit00** [comment]
Hey @murugappanrm 

Prisma runs only on the backend so you will need a way or a framework to actually run the Prisma calls on the server and fetch the data from the frontend. So I don't think you would be able to use Prisma in a vue app but rather you can call a backend which uses Prisma in your Vue application.

**Comment by murugappanrm** [comment, marked as answer]
Hi @pantharshit00 ,

Thank you for the assistance. I guessed the same too. I am a newbie to Vue world. The  reservations on the use of Prisma are these:

(1)  If it needs a back-end frame like Nuxt. I would rather create a headless CMS back-end using either Laravel or Codeigniter which are much easier to develop and maintain Vuejs as a clean front-end using Axios.
(2) The odd part is that it can only use one database whereas if I use Codeigniter or Laravel I could use as many databases as needed and also various types.

The hind-side to my reservations is that it need 2 languages. Oh well, just have to drop Prisma unless i am using Nuxt. Thank you once again.

**Reply by pantharshit00** [reply]
> (1) If it needs a back-end frame like Nuxt. I would rather create a headless CMS back-end using either Laravel or Codeigniter which are much easier to develop and maintain Vuejs as a clean front-end using Axios.

It totally upto you which stack is more comfortable for you. If you like PHP, you can choose to use it on the backend. Prisma is just a Node.js ORM just like Eloquent ORM which is shipped with Laravel. 


> (2) The odd part is that it can only use one database whereas if I use Codeigniter or Laravel I could use as many databases as needed and also various types.

You can also use multiple database here. You will just need to instantiate another instance of Prisma client. Prisma also supports a wide range of databases: https://www.prisma.io/docs/reference/database-reference/supported-databases

**Reply by murugappanrm** [reply]
@pantharshit00 

I guess Prisma is what ORM is to Laravel. Prisma needs Nuxt (or something) whereas Codeigniter does not. The only good thing about Prisma is it is for pure users of vuejs or nuxt who do not want other types of backend.  

Thank you for all the insight.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
